source/clang/package.d

```diff
@@ -852,3 +852,77 @@ struct Token {
         return kind == other.kind && spelling == other.spelling;
     }
 }
+
+/** A comment in the source text. */
+struct Comment {
+    CXComment x;
+
+    /**  What kind of comment is this? */
+    auto kind() {
+        return clang_Comment_getKind(x);
+    }
+
+    /**  Get this comment children comment */
+    auto get_children() {
+        return CommentChildrenIterator(x, clang_Comment_getNumChildren(x), 0);
+    }
+
+    /**  Given that this comment is the start or end of an HTML tag, get its tag name. */
+    auto get_tag_name() {
+        return toString(clang_HTMLTagComment_getTagName(x));
+    }
+
+    /**  Given that this comment is an HTML start tag index, get its attributes. */
+    auto get_tag_attrs() {
+        return CommentAttributesIterator(x, clang_HTMLStartTag_getNumAttrs(x), 0);
+    }
+}
+
+/**  An iterator for a comment children */
+struct CommentChildrenIterator {
```

Comment:
A TODO would be to use D ranges instead of this iterator which is based on the Rust code.

Reply:
Fixed in https://github.com/atilaneves/libclang/pull/25/commits/8e59d7155ef108ce4f5eeb8c4816fddf65fd6664